docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Apache Kafka Broker"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure the 
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
+
+This guide assumes a 
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component), only
+`eventing-core` is required.
+
+## Installation
+
+```bash
+kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+```
+
+Verify that `kafka-broker-controller`, `kafka-broker-receiver` and `kafka-broker-dispatcher` are running:
+
+```bash
+kubectl get deployments.apps -n knative-eventing 
+
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+eventing-controller            1/1     1            1           10s
+eventing-webhook               1/1     1            1           9s
+kafka-broker-controller        1/1     1            1           3s
+kafka-broker-dispatcher        1/1     1            1           4s
+kafka-broker-receiver          1/1     1            1           5s
+```
+
+## Create a Kafka Broker
+
+A Kafka Broker object looks like this:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: Kafka
+  name: default
+  namespace: default
+spec:
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing
+  # Optional dead letter sink, you can specify either:
+  #  - deadLetterSink.ref, which is a reference to a Callable
+  #  - deadLetterSink.uri, which is an absolute URI to a Callable (It can potentially be out of the Kubernetes cluster)
+  delivery: 
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: dlq-service
+```
+
+`spec.config` should reference any `ConfigMap` that looks like the following:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-broker-config
+  namespace: knative-eventing
+data:
+  # Number of topic partitions
+  default.topic.partitions: "10"
+  # Replication factor of topic messages.
+  default.topic.replication.factor: "1" 
+  # A comma separated list of bootstrap servers. (It can be in or out the k8s cluster)
+  bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092" 
+```
+
+The above `ConfigMap` is installed in the cluster, and you can edit
+the configurations or create a new one with the same shape 
+depending on your needs.
+
+## Set as default broker implementation
+
+To set the Kafka Broker as your default implementation you can edit
+the `config-br-defaults` `ConfigMap` in `knative-eventing` namespace.
+In this way, you can avoid specifying 
+`metadata.annotations.eventing.knative.dev/broker.class` or 
+`spec.config` for each Kafka Broker created in the cluster
+or in a specific namespace.
```

Comment:
```suggestion
To set the Kafka broker as the default implementation for all brokers in the Knative deployment, you can apply global settings by modifying the `config-br-defaults` ConfigMap in the `knative-eventing` namespace.

This allows you to avoid configuring individual or per-namespace settings for each broker, such as `metadata.annotations.eventing.knative.dev/broker.class` or 
`spec.config`.
```